### DISCUSSION TITLE
CameraView - Focus and Exposure

### DISCUSSION THREAD

**Discussion by GUOLDEV** [CommunityToolkit/Maui#171] [New Feature Discussions]
Implement camera controls such as a focus on tap, exposure, etc, like in the [MediaPlugin](https://github.com/jamesmontemagno/MediaPlugin) TakePhotoAsync or in the [ManualCameraControls](https://github.com/xamarin/ios-samples/tree/main/ManualCameraControls) as a reference.

![image](https://user-images.githubusercontent.com/22019245/119453550-e1fc4c00-bd7a-11eb-95fd-7e1d548e0421.png)

Implement camera properties:

- FocusMode;
```
public CameraFocusMode FocusMode { get; set; }

public enum CameraFocusMode
{
    Locked,
    AutoFocus,
    ContinuousAutoFocus
}
```
- ExposureMode;
```
public CameraExposureMode ExposureMode { get; set; }

public enum CameraExposureMode
{
    Locked,
    AutoExpose,
    ContinuousAutoExposure,
    Custom
}
```

**Comment by ashlar64** [comment]
I agree.  We need many of the features that is found in the MediaPlugin.

**Comment by CliffAgius** [comment]
Currently being worked on #259
